Sources/Application/Game/Player.swift

```diff
@@ -0,0 +1,22 @@
+public class Player {
+    public var name: String
+    public var order: Int
+    public var correctGuesses: [String]
+    public var incorrectGuesses: [String]
+
+    public init(name: String, order: Int, correctGuesses: [String], incorrectGuesses: [String]) {
+        self.name = name
+        self.order = order
+        self.correctGuesses = correctGuesses
+        self.incorrectGuesses = incorrectGuesses
+    }
+
+    public func appendGuess(word: String, guess: Guess) {
```

Comment:
I am going to say yes. The player is keeping track of his own guesses so the attributes `correctGuesses` and `incorrectGuesses` travel with the player. 

I know that's not the right answer though :-(